randovania/games/am2r/logic_database/The Depths.json

```diff
@@ -4205,6 +4205,44 @@
                                         "type": "template",
                                         "data": "Can Use Spider Ball"
                                     },
+                                    {
+                                        "type": "and",
+                                        "data": {
+                                            "comment": "Climb the right side and glide to the left https://youtu.be/Gq5wq0yHvCY",
```

Comment:
Dont apply suggestions to database files. You need to update them from within the editor, otherwise you're gonna miss updating the json too like here.

Reply:
This time I fixed it myself since I have a PR that's reliant on this.